### PR TITLE
fix #98 - adds Browser support section

### LINKED ISSUE #98: Add Browser Support Table to Psammead
**Is your feature request related to a problem? Please describe.**
We currently do not have any reference to our Browser support support list within Psammead.

We should add it to the Psammead Readme.md

**Describe the solution you'd like**
We would like to add a high-level summary of Browser Support support into Psammead, using this as an example:
https://github.com/alphagov/govuk-frontend#browser-support

@pjlee11 created a browser support matrix based on a 0.05% metric, and put this browser support matrix below:

| Browser           | Lowest version    |
| ----------------- | ---------- |
| Safari           | 7    |
| Facebook for iPhone           | 187    |
| Chrome for iOS           | 68    |
| Chrome           | 25    |
| Edge           | 14    |
| Firefox           | 48    |
| IE           | 9    |
| Amazon Silk           | 69    |
| Opera Mini           | 4    |
| Android Browser           | 4    |
| Nokia           | 6280    |
| Firefox for iOS           | 13    |
| Facebook for Android           | 190    |

We should add this matrix to the README, with a note that we don't actually support it yet but that this level of support is an aspiration for v1.

**Describe alternatives you've considered**
We could have the browser support in another file or another repository. 

**Testing notes**
[Tester to complete]

Dev insight: This is documentation. It should be easily understood by Devs, Testers, Product Owners, UX Designers & Business Analysts. 

**Context**
Analysis of existing documentation and resources:

- We have Browser stats for the News site, but they're not granular enough for our purposes https://docs.google.com/spreadsheets/d/1X6e7_3XCtUN9SFsMrSFCUMbl6KOjkTdS6Su9RcAJlBc/edit 
- We have a more detailed breakdown of browser stats that are exported from Comscore into a spreadsheet - `browsers-2018-08-09.xlsx`  (File removed, due to reasoning here: https://github.com/BBC-News/psammead/issues/98#issuecomment-444129534)

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by ChrisBAshton**
Resolves #98

adds Browser support section

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by sareh — DISMISSED** [review]
👍 Looks great

File: README.md [hunk 1]
```diff
@@ -125,10 +125,30 @@ Our recommended `.babelrc` config for this is here:
 
 [See documentation on the Styled Components site](https://www.styled-components.com/docs/tooling#babel-plugin)
 
-## Browser support
+## Support levels
```
Comment (dr3) [inline README.md]: 🙈 would be nice to get some emoji down here

Comment (bcmn) [reply README.md]: 📊

**Review by bcmn — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (ChrisBAshton) [reply README.md]:
```suggestion
## :bar_chart: Support levels
```

**Review by bcmn — APPROVED** [review]
Excellent emoji work 👍

**Review by sareh — APPROVED** [review]
📈 Looks great! :)

**Pull request merged**